src/nvim/lua/executor.c

```diff
@@ -252,6 +252,52 @@ static int nlua_luv_cfpcall(lua_State *lstate, int nargs, int nresult,
   return retval;
 }
 
+static void nlua_luv_thread_error_event(void **argv)
+{
+  char *error = (char *)argv[0];
+  msg_ext_set_kind("lua_error");
+  emsgf_multiline("Error occuring in luv thread:\n%s", error);
```

Comment:
more specific: "error occuring in luv callback in thread"

Reply:
also don't need "occuring"

```suggestion
  emsgf_multiline("Error in luv callback thread:\n%s", error);
```

Reply:
Done.